at-luantran/Week3/2018_07_05_exercise3_rubyday3.rb

```diff
@@ -0,0 +1,43 @@
+class HtmlToText
+  def get_html(html)
+    arr_char = html.split('')
+    @string = ''
+    stack = []
+    i = 0
+    check = false
+    while i < arr_char.length do
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/838e41b2d4bb928094b6df310b810687d65450b5/at-luantran/Week3/2018_07_05_exercise3_rubyday3.rb#L8"></span>Style/WhileUntilDo: Do not use <code>do</code> with multi-line <code>while</code>. (https://github.com/rubocop-hq/ruby-style-guide#no-multiline-while-do)</td>
    </tr>
  </tbody>
</table>